spec/datadog/appsec/configuration/settings_spec.rb

```diff
@@ -757,5 +757,45 @@ def patcher
         end
       end
     end
+
+    describe 'standalone' do
+      describe '#enabled' do
+        subject(:enabled) { settings.appsec.standalone.enabled }
+
+        context 'when DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED' do
+          around do |example|
+            ClimateControl.modify('DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED' => appsec_standalone_enabled) do
+              example.run
+            end
+          end
+
+          context 'is not defined' do
+            let(:appsec_standalone_enabled) { nil }
+
+            it { is_expected.to eq false }
+          end
+
+          context 'is defined' do
+            let(:appsec_standalone_enabled) { 'true' }
+
+            it { is_expected.to eq(true) }
+          end
+        end
+      end
+
+      describe '#enabled=' do
+        subject(:set_appsec_standalone_enabled) { settings.appsec.standalone.enabled = appsec_standalone_enabled }
+
+        [true, false].each do |value|
+          context "when given #{value}" do
+            let(:appsec_standalone_enabled) { value }
+
+            before { set_appsec_standalone_enabled }
+
+            it { expect(settings.appsec.standalone.enabled).to eq(value) }
+          end
+        end
```

Comment:
It might be personal opinion, but I think DRY in tests makes them harder to read. WDYT?

Reply:
I agree for more complex cases, but I think that boolean are simple enough that it does not lower the readability too much. `[true, false].each` is used by APM team (e.g. in `spec/datadog/core/configuration/settings_spec.rb`) so I think it's better to use it for cohesion.

Reply:
To be honest, this spec is such a massive copy-paste with these boolean checks that I kinda wish we had a `behaves_like_a_binary_setting` or something like that helper rather than repeating it every time.

Not to say we need to fix it in this PR, but yeah, it's been bugging me for a while and it seemed like a good time to share ;)

Reply:
Yeah, shared behavior sounds good. Agree that maybe not right now, but it's good to share the pain 😉

Reply:
what are we actually testing here? If we want to test that the value is being assigned, it is enough to test once with one value. If we want to test that this setting could be only set to a boolean value, we also need a test case when a non-boolean value is being passed to this setting.

Reply:
We are testing that the value is being assigned. I agree that these tests should be reworked (on another PR)